bitnami/keycloak/templates/statefulset.yaml

```
@@ -72,6 +72,9 @@ spec:
       {{- if .Values.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
+      {{- if .Values.serviceLinks.disabled }}
+      enableServiceLinks: false
+      {{- end }}
```

`enableServiceLinks` is only available in Kubernetes 1.13 and later, to keep backward compatibility, I suggest the following change:
```suggestion
      {{- if semverCompare ">= 1.13" (include "common.capabilities.kubeVersion" .) }}
      enableServiceLinks: {{ .Values.enableServiceLinks }}
      {{- end }}
```